src/main/jhipster/src/main/webapp/app/shared/auth/user-route-access-service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@angular/core';
+import { ActivatedRouteSnapshot, CanActivate, Router, RouterStateSnapshot } from '@angular/router';
+import { Principal } from './principal.service';
+
+@Injectable()
+export class UserRouteAccessService implements CanActivate {
```

Comment:
almost identical to route-access.service.ts, that one should work

Reply:
I will remove UserRouteAccessService and use RouteAccessService instead.